ass-02.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*ID: 602110195
+Name: Zhang Hao(Henry)
+Wechat: hikki*/
+$XX=substr($_SERVER['argv'][1],0,2);
+$ss=substr($_SERVER['argv'][1],2);
+$m=0;
+for($i=0;;$i++){
+    if(substr($ss,$i,1)==0){
+        $m++;
+    }else{
+        break;
+    }$sss=substr($ss,$m);
+}$f=fopen($_SERVER['argv'][2],'r');
+fscanf($f,"%d",$n);
+$products=[];
+for($i=0;$i<$n;$i++){
+    $product=[];
+    $product['product']=fgets($f,4096);
+    $product['ss']=$sss+$i;
+    $product['0']=str_repeat("0",$m);

```

Comment:
If you use strlen() on previous lines.
```suggestion
    $product['ss']= str_pad($sss+$i, $m, "0", STR_PAD_LEFT);

```